.github/workflows/release.yml

```diff
@@ -21,18 +21,19 @@ jobs:
       - name: "Build release apk"
         run: ./gradlew assembleRelease --stacktrace
 
-      - name: "Sign release"
-        uses: r0adkll/sign-android-release@v1
+      - uses: ilharp/sign-android-release@v1 # Or use @nightly
+        name: Sign app APK
         id: sign_app
         with:
-          releaseDirectory: app/build/outputs/apk/release
-          signingKeyBase64: ${{ secrets.SIGNING_KEY }}
-          alias: ${{ secrets.ALIAS }}
-          keyStorePassword: ${{ secrets.KEY_STORE_PASSWORD }}
-          keyPassword: ${{ secrets.KEY_PASSWORD }}
+          releaseDir: app/build/outputs/apk/release
+          signingKey: ${{ secrets.ANDROID_SIGNING_KEY }}
+          keyAlias: ${{ secrets.ANDROID_KEY_ALIAS }}
+          keyStorePassword: ${{ secrets.ANDROID_KEYSTORE_PASSWORD }}
+          keyPassword: ${{ secrets.ANDROID_KEY_PASSWORD }}
+          buildToolsVersion: 33.0.0
 
       - name: "Rename archive"
-        run: mv ${{ steps.sign_app.outputs.signedReleaseFile }} app-release.apk
+        run: mv ${{steps.sign_app.outputs.signedFile}} app-release.apk
```

Comment:
As you already know, I'm not going to continue maintaining this. So if you want to change the release process, please fork/copy the repo and I'll link yours

Reply:
I am still fighting with it not signing. So I am as a java noib probably nit the best candidate to maintain this, but I will try to get this release built, and theoretically having a small gh org like oldpipe community, where all can be collabs/members is a valid idea too. Between all the people that are using this currently, I assume tens of them.

Reply:
I split it, so this can be merged here. I will try and solve that issue with signing in another separate branch. Thanks :)

Reply:
> As you already know, I'm not going to continue maintaining this. So if you want to change the release process, please fork/copy the repo and I'll link yours

Hey! I managed to sign the apk and do the release. Wow, this feels cool :D

Reply:
@kubo6472 Do you then want to take over creating releases? I will then link your project

Reply:
> @kubo6472 Do you then want to take over creating releases? I will then link your project

Hey, let me move it to an org, and we'll see. You could link it any time in the future, too. As I said, I'm not sure I can take this on myself.